invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/page.html

```diff
@@ -0,0 +1,18 @@
+{#
+    Copyright (C) 2023 CERN.
+
+    Invenio App RDM is free software; you can redistribute it and/or modify it
+    under the terms of the MIT License; see LICENSE file for more details.
+  #}
+
+{%- extends "invenio_theme/page.html" %}
+
+{%- block body %}
+  {{ super() }}
+
+  {%- block javascript %}
+    {{ super() }}
+    {{ webpack['overridable-registry.js'] }}
```

Comment:
you could just add the sctipt to `/invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/theme.js`, so you won't need to modify the template and BASE_TEMPLATE config.
Alternatively, it can be placed under `invenio-app-rdm/invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/javascript.html`, the template which holds all the attached js assets